source/constants/blueprints.py

```diff
@@ -4,6 +4,7 @@
 INDEX_BLUEPRINT_NAME: Final[str] = "index"
 GAME_STATS_BLUEPRINT_NAME: Final[str] = "game_stats"
 PROFILE_BLUEPRINT_NAME: Final[str] = "profile"
+REPORTS_TABLE_NAME: Final[str] = "report"
```

Comment:
Just a small tweak here, this constant should be named something as "REPORT_BLUEPRINT_NAME" instead of "REPORTS_TABLE_NAME" since this is the `blueprints.py` file.